web/site/playwright.config.ts

```diff
@@ -15,6 +15,7 @@ const devicesToTest = [
 ] satisfies Array<string | typeof devices[string]>
 
 export default defineConfig<ConfigOptions>({
+  globalSetup: './tests/e2e/utils/auth-setup',
```

Comment:
this will run when the test runner boots up and saves the user data to a json file to be reused